client/containers/App.js

```diff
@@ -12,7 +12,16 @@ const mapDispatchToProps = (dispatch) => ({
     dispatch(fetchCategories());
     dispatch(fetchDiscounts());
     dispatch(fetchFilters());
-    dispatch(fetchUsername());
+
+    const cookies = {};
```

Comment:
@IgorChaikin I think you have already implemented similar functionality in `tests/cookiesParser.js`, probably you could move it to some `utils` directory and reuse here (Don't repear yourself principle)